app/models/organization.rb

```diff
@@ -9,7 +9,8 @@ class Organization < ApplicationRecord
 
   belongs_to :business_plan
   belongs_to :contact_person, class_name: 'User', optional: true
-  has_many :users, class_name: 'User', dependent: :destroy
+  has_many :users_organizations, dependent: :destroy
+  has_many :users, through: :users_organizations, dependent: :destroy
```

Comment:
Would that destroy a user if the organization is destroyed? I think we should keep the user as it can also belong to other organizations. The standard behavior of this should be to do nothing on the user itself and just destroy the relation.